spec/system/user_uses_the_editor_spec.rb

```diff
@@ -22,6 +22,16 @@ def fill_markdown_with(content)
     within("#article-form") { fill_in "article_body_markdown", with: content }
   end
 
+  describe "Viewing the editor", js: true do
+    it "renders the logo_svg or Community name as expected" do
+      visit "/new"
+      expect(page).to have_css(".site-logo")
+      within(".truncate-at-2") do
+        expect(page).to have_text("DEV(local)")
```

Comment:
I observed that the spec renders the `community_name` rather than the logo, so this test ensures that the `.site_logo` CSS is on the page, in addition to the truncated `community_name` that is actually rendered in the test.